### PR TITLE
Script for installing node

### PR DESCRIPTION
**Pull request by IeSua-lab**
Script for installing node. Significantly speed up the process of installing a node.

Script for installing and configuring Joystream nodes.
Works as root and user.

Only 3 lines:
1. Download joystream-node-install.sh
2. chmod +x ./joystream-node-install.sh
3. ./joystream-node-install.sh

The script will install the node. At the end of the installation, it will ask you what to name your node.

GIF file (how it works):
https://imgur.com/a/lliuWZk

### CODE REVIEW

**Comment by IeSua-lab** [comment]
> Hey @IeSua-lab
> This looks good to me!
> 
> If you get it tested and approved by the Council, I'll merge it :)
> 
> So sorry for the late reply - will send some tokens!

Hello.
Proposal has already been approved by the Council - https://testnet.joystream.org/#/proposals/172
I wrote about this in the report for Scoring Period 1 (Member ID: 1497, Member handle: wasabi).

I tested it many times and gave links in the Russian telegram group - https://t.me/JoystreamRussian/10044
People said "great, thanks", but no one said whether they tested the script or not. So, I think you need to do it yourself or give instructions to the council.

File: community-contributions/scriptnodeSetup/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Installation with script
+
+Login as root, then do:
+```
+wget https://raw.githubusercontent.com/IeSua-lab/community-repo/scriptnodeSetup/community-contributions/scriptnodeSetup/joystream-node-install.sh
```
Comment (bwhm) [inline community-contributions/scriptnodeSetup/README.md:5]: Could you remove this link, or change to this repo instead? I'm a little reluctant to have people to run a script that may be changed :)

**Review by bwhm — CHANGES_REQUESTED** [review]
Great!

Only the one comment, and I'll merge :)

File: community-contributions/scriptnodeSetup/README.md (hunk 1 above)
Comment (IeSua-lab) [reply community-contributions/scriptnodeSetup/README.md]: Removed the link. Changed to "Download joystream-node-install.sh"
Is that okay?

**Review by bwhm — APPROVED** [review]

**Pull request merged**